lib/core/base/virtual-node/serial-virtual-node.js

```diff
@@ -84,3 +84,5 @@ function normaliseAttrs({ attributes = {} }) {
 		return attrs;
 	}, {});
 }
+
+axe.SerialVirtualNode = SerialVirtualNode;
```

Comment:
We shouldn't need to add this to the axe namespace. All our files are loaded directly into the test, so `SerialVirtualNode` should be available as a global, just like our `Audit` is in `test/core/base/audit.js`

Reply:
That's not correct. `core` and `commons` are in an isolated scope. Only files in `test/core` have access to the scope of `src/core`. Files in `test/commons`, `tests/checks` and `tests/integrations` don't have access. Same with `src/commons`. Only files in `tests/commons` have access to the `src/commons` scope.

Axe-core isn't just one big global scope.

Reply:
Ah I didn't know that. Other than for the test, would there be a reason to have 'VirtualNode' and 'SerialVirtualNode' on the public API?